scout/adapter/mongo/variant.py

```diff
@@ -191,8 +191,6 @@ def variants(
             case_id, query=query, variant_ids=variant_ids, category=category
         )
         sorting = []
-        if sort_key == "variant_rank":
-            sorting = [("variant_rank", pymongo.ASCENDING)]
```

Comment:
Perhaps pymongo.DESCENDING instead, as we want the ones with higher numbers before?